### PR TITLE
Jan pre dev

### PR DESCRIPTION
**Pull request by JanProgrammierung**
- Major theme and color update
- Change of header text location to content.ts

### CODE REVIEW

**Comment by JanProgrammierung** [comment]
Added some new commits

File: pmp-frontend-app/src/components/ContactFormComponent.tsx [hunk 1]
```diff
@@ -122,7 +122,7 @@ export default function ContactFormComponent(): ReactElement {
                             (<p>
                                 {(messageCharLimit - inputFields.message.length) + " characters left"}
                             </p>) : 
-                            <p className="error text-red-400">
+                            <p className="error text-error-content">
```
Comment (SevLG) [inline pmp-frontend-app/src/components/ContactFormComponent.tsx:125]: I saw that error-content is white in our tailwind config. Is the error text visible with white text? If not we could try "error bg-error text-error-content" I think right?

Comment (JanProgrammierung) [reply pmp-frontend-app/src/components/ContactFormComponent.tsx:125]: I see, thanks for spotting! I suggest we just use error text-error. Then the font will be red.
I change it quickly

**Comment by JanProgrammierung** [comment]
@SevLG new review required

File: pmp-frontend-app/src/components/ContactFormComponent.tsx [hunk 2]
```diff
@@ -136,12 +136,12 @@ export default function ContactFormComponent(): ReactElement {
                                 <input type="submit" value="Submit" className="btn btn-wide bg-fuchsia-950 text-white hover:bg-fuchsia-800 active:bg-fuchsia-900 focus:outline-none focus:ring focus:ring-fuchsia-300" />
                                 :
                                 <>
-                                    <p className="error text-red-400">Please tick 'I'm not a robot' above the 'Submit' button.</p>
-                                    <div className='btn btn-wide bg-zinc-300 text-gray-500 cursor-not-allowed'>Submit</div>
+                                    <p className="error text-error-content">Please tick 'I'm not a robot' above the 'Submit' button.</p>
```
Comment (SevLG) [inline pmp-frontend-app/src/components/ContactFormComponent.tsx:139]: Sorry, forgot to mention error-content is also used down here

**Comment by JanProgrammierung** [comment]
@SevLG Hopefully final review now

**Review by SevLG — APPROVED** [review]

**Pull request merged**